CHANGELOG.md

```diff
@@ -7,6 +7,31 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+This update features a big rewrite into a component-based API. The core concepts remain the same, but are now expressed through ECS. It's recommended to revisit the quick start guide.
+
+### Changed
+
+- Input contexts are now components.
+- Actions for contexts are now represented by entities with an `ActionOf<C>` relation, where `Actions<C>` is the target located on the context entity (which now only stores entities).
```

Comment:
This sentence isn't clear to me as a reader. "is the target" is particularly unclear to me, and it would be nice to introduce "context entities" in advance.